Parser/pegen/pegen.c

```diff
@@ -0,0 +1,1834 @@
+#include <Python.h>
```

Comment:
Would you mind to add `Parser/pegen` subdirectory to `SRCDIRS` in `configure.ac`? So `make tags` will also parse `Parser/pegen/*.c` and `Parser/pegen/*.h`.

Reply:
Done.